django/contrib/admin/options.py

```diff
@@ -109,9 +109,8 @@ class BaseModelAdmin(six.with_metaclass(forms.MediaDefiningClass)):
     show_full_result_count = True
     checks_class = BaseModelAdminChecks
 
-    @classmethod
-    def check(cls, model, **kwargs):
-        return cls.checks_class().check(cls, model, **kwargs)
+    def check(self, **kwargs):
+        return self.checks_class().check(self, **kwargs)
```

Comment:
My main argument for running checks against `ModelAdmin` instances was that they needed to be bound to a `model` to make any sense. I just realized `InlineModelAdmin` checks also require reference to a `parent_model` so I guess checks should also be run against instances here to. I retract my comment.